source/documentation/data-docs/amazon-s3.md

```diff
@@ -34,7 +34,7 @@ You can only create new warehouse data sources in the Analytical Platform contro
 
 To create a new warehouse data source:
 
-1.  Go to the Analytical Platform [control panel](https://controlpanel.services.alpha.mojanalytics.xyz/).
+1.  Go to the Analytical Platform [control panel](https://controlpanel.services.analytical-platform.service.justice.gov.uk/).
```

Comment:
maybe in this section we can have both links - one for old platform and one for new? Maybe say something like go to the old link unless you know you've been upgraded or have a new account in which case go to this link.